openscap_report/scap_results_parser/data_structures/rule.py

```diff
@@ -18,16 +15,16 @@ class Rule:  # pylint: disable=R0902
     multi_check: bool = False
     time: str = ""
     severity: str = ""
-    identifiers: list[Identifier] = field(default_factory=list)
-    references: list[Reference] = field(default_factory=list)
+    identifiers: list = field(default_factory=list)
```

Comment:
According to https://docs.python.org/3.8/library/typing.html, you can have a typing hint of list of strings in Python 3.8, but you have to use the typing module, i.e. `typing.List[...]` not the `list[...]` syntax.